app/routes/auth/logout.js

```diff
@@ -12,7 +12,7 @@ export default class AuthLogoutRoute extends Route {
         let wasMockLoginSession = this.session.isMockLoginSession;
         await this.session.invalidate();
         let logoutUrl = wasMockLoginSession
-          ? this.router.urlFor('mock-login')
+          ? this.router.urlFor('login')
```

Comment:
If the session was a mock session wouldn't it need to redirect to mock-login

Reply:
@lagartoverde I changed it back to mock-login.